docs/src/test/scala/com/lightbend/lagom/docs/ServiceSupport.scala

```diff
@@ -3,36 +3,41 @@ package com.lightbend.lagom.docs
 import java.util.concurrent.CompletionStage
 
 import com.google.inject.AbstractModule
-import com.lightbend.lagom.javadsl.api.transport.{ RequestHeader, ResponseHeader }
-import com.lightbend.lagom.javadsl.api.{ ServiceCall, ServiceLocator, Service}
-import com.lightbend.lagom.javadsl.server.{ HeaderServiceCall, ServiceGuiceSupport }
-import org.scalatest.{ Matchers, WordSpecLike }
-import play.api.Application
+import com.lightbend.lagom.javadsl.api.transport.{RequestHeader, ResponseHeader}
+import com.lightbend.lagom.javadsl.api.{Service, ServiceCall, ServiceLocator}
+import com.lightbend.lagom.javadsl.server.{HeaderServiceCall, ServiceGuiceSupport}
+import org.scalatest.{Matchers, WordSpecLike}
+import play.api.{Application, Configuration}
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.inject.bind
 import play.core.server.Server
 
-import scala.concurrent.{ Future, Promise }
+import scala.concurrent.{Future, Promise}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.reflect.ClassTag
 import scala.compat.java8.FutureConverters._
 import com.lightbend.lagom.internal.testkit.TestServiceLocator
 import docs.services.test.ServiceTestModule
 import com.lightbend.lagom.internal.testkit.TestServiceLocatorPort
 import com.lightbend.lagom.javadsl.persistence.jdbc.JdbcPersistenceModule
-import play.api.db.{ DBModule, HikariCPModule }
+import com.typesafe.config.ConfigFactory
+import play.api.db.{DBModule, HikariCPModule}
 
 trait ServiceSupport extends WordSpecLike with Matchers {
 
   def withServer[T](applicationBuilder: GuiceApplicationBuilder = new GuiceApplicationBuilder())(block: Application => T): T = {
     val port = Promise[Int]()
     val testServiceLocatorPort = TestServiceLocatorPort(port.future)
 
-    val application = applicationBuilder
-      .bindings(bind[TestServiceLocatorPort].to(testServiceLocatorPort))
-      .overrides(bind[ServiceLocator].to(classOf[TestServiceLocator]))
-      .disable(classOf[ServiceTestModule]) // enabled in application.conf
-      .build()
+    val application =
+      applicationBuilder
+        .loadConfig { env =>
```

Comment:
You can also use:

```scala
applicationBuilder
  .configure("lagom.cluster.bootstrap.enabled" -> "off")
```

Reply:
hmm, I saw it, but thought it was returned `Configuration` instead. I will change it, better when shorter.